lib/libc/net/lib_inetaddr.c

```diff
@@ -88,7 +86,7 @@ in_addr_t inet_addr(FAR const char *cp)
 	uint32_t result;
 
 	if (!isdigit((char)*cp)) {
-		return (IPADDR_NONE);
+		return ((unsigned int)0xffffffffUL);
```

Comment:
Let's choose one way
1. include lwip header and use a definition which is defined in lwip header
2. use this value directly without including lwip header